flink-filesystems/flink-s3-fs-base/pom.xml

```diff
@@ -108,83 +107,6 @@ under the License.
 					<skip>true</skip>
 				</configuration>
 			</plugin>
-
-			<!-- Relocate all S3 related classes -->
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
```

Comment:
without the shade configuration this module in itself shouldn't exist; and should instead be a parent module for the other S3 filesystems containing the current dependencies.

Reply:
Btw this shade configuration caused [FLINK-14975](https://issues.apache.org/jira/browse/FLINK-14975). This configuration shaded guava 11 and presto either overwrote it by guava 21 (on travis) or not (on my machine).

Reply:
Couldn't get the parent pom to work as the module contains quite a bit of shared code (e.g., AbstractS3FileSystemFactory and multi upload stuff). Can we leave it as is?

Reply:
I didn't consider the module containing classes. Let's leave it as is; without any shade-plugin configuration